src/decision/CriteriaWeights.jl

```diff
@@ -1,28 +1,51 @@
-Base.@kwdef struct CriteriaWeights{P,N} <: EcoModel
-    wave_stress::P = Param(
+Base.@kwdef struct CriteriaWeights{P,P2,P3,N} <: EcoModel
+    seed_wave_stress::P = Param(
         1.0;
         ptype="real",
         bounds=(0.0, 1.0),
         default_bounds=(0.0, 1.0),
         dists="unif",
-        name="Wave Stress",
-        description="Importance of avoiding wave stress. Higher values places more weight on areas with low wave stress.",
+        criteria_keywords=(:seed, :weight),
+        name="Seed Wave Stress",
+        description="Importance of avoiding wave stress when seeding. Higher values places more weight on areas with low wave stress.",
     )
-    heat_stress::P = Param(
+    seed_heat_stress::P = Param(
         1.0;
         ptype="real",
         bounds=(0.0, 1.0),
         default_bounds=(0.0, 1.0),
         dists="unif",
-        name="Heat Stress",
-        description="Importance of avoiding heat stress. Higher values places more weight on areas with low heat stress.",
+        criteria_keywords=(:seed, :weight),
+        name="Seed Heat Stress",
+        description="Importance of avoiding heat stress when seeding. Higher values places more weight on areas with low heat stress.",
+    )
+    shade_wave_stress::P = Param(
+        1.0;
+        ptype="real",
+        bounds=(0.0, 1.0),
+        default_bounds=(0.0, 1.0),
+        dists="unif",
+        criteria_keywords=(:fog, :weight),
+        name="Shade Wave Stress",
+        description="Importance of avoiding wave stress when fogging. Higher values places more weight on areas with low wave stress.",
+    )
+    shade_heat_stress::P = Param(
+        1.0;
+        ptype="real",
+        bounds=(0.0, 1.0),
+        default_bounds=(0.0, 1.0),
+        dists="unif",
+        criteria_keywords=(:fog, :weight),
+        name="Shade Heat Stress",
+        description="Importance of avoiding heat stress when fogging. Higher values places more weight on areas with low heat stress.",
     )
```

Comment:
Description and criteria keywords mention `fog`, but factor and field names use "shade" ?

Reply:
Yeah, I originally had "shade" as the descriptor but really it's fogging so I changed it. I can change it back to shade for consistency, or I can change the parameters labeled with "shade" to be labeled with "fog", which will require some other changes throughout the code. What do you think?

Reply:
Which is more specific to what is being represented?

Reply:
"fog" is more relevant, I can change it to that. Is it ok to also change the order of some parameter names in this PR while I'm at it? Some have seed/shade at the end, some at the start etc.

Reply:
Yes?